SPECS/environment-modules/environment-modules.spec

```
@@ -87,108 +90,91 @@ touch %{buildroot}%{_sysconfdir}/profile.d/modules.{csh,sh}
   --slave %{_sysconfdir}/profile.d/modules.csh modules.csh %{_datadir}/Modules/init/profile.csh \
   --slave %{_bindir}/modulecmd modulecmd %{_datadir}/Modules/libexec/modulecmd.tcl \
-  --slave %{_mandir}/man1/module.1.gz module.1.gz %{_mandir}/man1/module-c.1.gz \
-  --slave %{_mandir}/man4/modulefile.4.gz modulefile.4.gz %{_mandir}/man4/modulefile-c.4.gz
-
-%post compat
-%{_sbindir}/update-alternatives \
-  --install %{_sysconfdir}/profile.d/modules.sh modules.sh %{_datadir}/Modules/init/profile-compat.sh 10 \
-  --slave %{_sysconfdir}/profile.d/modules.csh modules.csh %{_datadir}/Modules/init/profile-compat.csh \
-  --slave %{_bindir}/modulecmd modulecmd %{_datadir}/Modules/libexec/modulecmd-compat
-
 %postun
 if [ $1 -eq 0 ] ; then
   %{_sbindir}/update-alternatives --remove modules.sh %{_datadir}/Modules/init/profile.sh
 fi
 
-%postun compat
-if [ $1 -eq 0 ] ; then
-  %{_sbindir}/update-alternatives --remove modules.sh %{_datadir}/Modules/init/profile-compat.sh
-fi
-
-
 %files
 %license COPYING.GPLv2
-%doc ChangeLog README NEWS.txt MIGRATING.txt CONTRIBUTING.txt diff_v3_v4.txt example.txt
+%doc ChangeLog README NEWS.txt MIGRATING.txt CONTRIBUTING.txt INSTALL.txt changes.txt
 %{_sysconfdir}/modulefiles
 %ghost %{_sysconfdir}/profile.d/modules.csh
 %ghost %{_sysconfdir}/profile.d/modules.sh
 %ghost %{_bindir}/modulecmd
 %{_bindir}/envml
-%{_libdir}/libtclenvmodules.so
+%dir %{_libdir}/%{name}
+%{_libdir}/%{name}/libtclenvmodules.so
 %dir %{_datadir}/Modules
 %{_datadir}/Modules/bin
 %dir %{_datadir}/Modules/libexec
 %{_datadir}/Modules/libexec/modulecmd.tcl
 %dir %{_datadir}/Modules/init
 %{_datadir}/Modules/init/*
+# do not need to require shell package as we "own" completion dir
+%dir %{_datadir}/bash-completion/completions
+%{_datadir}/bash-completion/completions/module
+%{_datadir}/bash-completion/completions/ml
+%dir %{_datadir}/zsh/site-functions
+%{_datadir}/zsh/site-functions/_module
+%dir %{_datadir}/fish/vendor_completions.d
+%{_datadir}/fish/vendor_completions.d/module.fish
 %dir %{_sysconfdir}/%{name}
 %config(noreplace) %{_sysconfdir}/%{name}/initrc
 %config(noreplace) %{_sysconfdir}/%{name}/modulespath
 %config(noreplace) %{_sysconfdir}/%{name}/siteconfig.tcl
 %{_datadir}/Modules/modulefiles
 %{_datadir}/modulefiles
-%ghost %{_mandir}/man1/module.1.gz
-%ghost %{_mandir}/man4/modulefile.4.gz
-%{_mandir}/man1/module-c.1.gz
-%{_mandir}/man4/modulefile-c.4.gz
+%{_mandir}/man1/ml.1.gz
+%{_mandir}/man1/module.1.gz
+%{_mandir}/man4/modulefile.4.gz
 %{macrosdir}/macros.%{name}
-
-
-%files compat
-%doc ChangeLog-compat NEWS-compat
-%{_datadir}/Modules/libexec/modulecmd-compat
-%{_mandir}/man1/module-compat.1.gz
-%{_mandir}/man4/modulefile-compat.4.gz
-
+%dir %{_datadir}/Modules/nagelfar
+%{_datadir}/Modules/nagelfar/*
 
 %changelog
+* Fri Feb 03 2023 Riken Maharjan <rmaharjan@microsoft.com> - 5.2.0-1
+- Move from extended to Core.
+- Update to 5.2.0 (from Fedora 38(license: MIT)).
```

I missed one more.;)

```suggestion
- Update to 5.2.0 (from Fedora 38 (license: MIT)).
```